hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/VolumeUsage.java

```diff
@@ -61,6 +61,14 @@ public long getUsedSpace() {
     return source.getUsedSpace();
   }
 
+  public void incrementUsedSpace(long usedSpace) {
+    source.incrementUsedSpace(usedSpace);
+  }
+
```

Comment:
Variable name should reclaimedSpace or so?

Reply:
That sounds like a good idea. I'll update the patch.